src/tests/ftest/daos_test/daos_core_test-rebuild.yaml

```diff
@@ -57,11 +57,12 @@ daos_tests:
   num_replicas:
     num_replicas: 1
   Tests: !mux
-    test_r_0-10:
-      daos_test: r
-      test_name: rebuild tests 0-10
-      args: -s3 -u subtests="0-10"
-      test_timeout: 1500
+# Skipped for DAOS-5851
+#    test_r_0-10:
+#      daos_test: r
+#      test_name: rebuild tests 0-10
+#      args: -s3 -u subtests="0-10"
```

Comment:
It looks like only subtest 5 is failing so my only question is could we do the following?
```suggestion
#      args: -s3 -u subtests="0-4,6-10"
```